Documentation/gettingstarted/k8s-install-rke.rst

```diff
@@ -0,0 +1,85 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _rke_install:
+
+********************************************
+Installation Using Rancher Kubernetes Engine
```

Comment:
```suggestion
Installation using Rancher Kubernetes Engine
```

One one hand, big `U` is more consistent on this page; On the other, small `u` makes it more consistent with its siblings:

![pic2](https://user-images.githubusercontent.com/17001771/110927262-53e8fb80-831d-11eb-9741-1829bc7332c7.png)

So no strong opinion here, use whatever you prefer.

Reply:
I also don't care whether we are capitalizing or not, but let's at least make it look consistent?